src/inquirer/themes.py

```diff
@@ -100,6 +100,20 @@ def __init__(self):
         self.List.unselected_color = term.normal
 
 
+class RedComposure(Default):
```

Comment:
Let's give it a unique name. How about this:

```suggestion
class RedSolace(Default):
```